File: its/projects/TestOnlyProject/UnitTest1.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace TestOnlyProject
{
    [TestClass]
    public class UnitTest1
    {
        [TestMethod]
        public void TestMethod1()
        {
            Assert.AreEqual(10, 5 + 5);
        }

        [TestMethod]
        [Ignore]
        public void IgnoredWithTestAndAllScopeRuleIssues(bool b)
        {
            if (b == true)
            {
                // Do something.
            }

            Assert.IsTrue(b);
        }
    }
}

File: its/projects/VbTestOnlyProjectTest/MyLib.Tests/UnitTest1.vb
Imports System.Text
Imports Microsoft.VisualStudio.TestTools.UnitTesting

<TestClass()> Public Class UnitTest1

    <TestMethod()> Public Sub TestMethod1()
    End Sub

    <TestMethod()>
    <Ignore>
    Public Sub IgnoredWithTestAndAllScopeRuleIssues(B As Boolean)

        If B = True Then
           ' Do something.
        End If

        Assert.IsTrue(B)
    End Sub

End Class
